src/index.html

```diff
@@ -2,7 +2,6 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
-  <link rel="stylesheet" href="styles.css">
```

Comment:
Thanks be to Hannah for helping us out with error messages due to this line and the one at the bottom of the file.